integration_tests/schemas/schemas.stacspec.org/1.0.0/item-spec/json-schema/itemcollection.json

```diff
@@ -30,7 +30,7 @@
                         "stac_version": {
                             "title": "STAC version",
                             "type": "string",
-                            "const": "1.0.0"
+                            "const": "0.9.0"
```

Comment:
This looks like it's going the wrong way. Can you confirm it's right please.

Reply:
Fixed